docs/source/intro-clx-loda-anomaly-detection.ipynb

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
<p>For each row we can examine the importance of each of the 5 features' contribution to the score. 0 is no importance and 1 is high importance.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>In the original NB there is an example with real dataset from shuttle, may be we can refer to it from this.</p>